atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/config/TargetedSweepInstallConfig.java

```diff
@@ -68,6 +68,15 @@ public int thoroughThreads() {
         return AtlasDbConstants.DEFAULT_TARGETED_SWEEP_THREADS;
     }
 
+    /**
+     * The number of background threads dedicated to running targeted sweep of tables with SweepStrategy NONE. This
+     * value should almost always be 0 or 1.
+     */
+    @Value.Default
+    public int noneThreads() {
+        return 1;
```

Comment:
I guess we should mention that if must be non-zero if there are tables that are not being swept.